RabbitMq/1.0.0/installer-manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "add-array-option": [
+        {
+            "target": "\\Pyz\\Client\\RabbitMq\\RabbitMqConfig::getSynchronizationQueueConfiguration",
+            "source": "Spryker\\Shared\\AssetExternalStorage\\AssetExternalStorageConfig::ASSET_EXTERNAL_SYNC_STORAGE_QUEUE"
```

Comment:
And we have a ticket where we should rename `AssetExternalStorageConfig` to `AssetStorageConfig` and other modules related to Assets: https://spryker.atlassian.net/browse/APPS-3066

Reply:
I think we should do this as a separate refactoring ticket after renaming ticket is done. Also, I'm not sure if this manual addition of the manifest is still relevant after creating the appropriate strategy in the release app.